crates/metrics/src/buckets.rs

```diff
@@ -76,6 +62,44 @@ fn initialize_buckets() -> HashMap<Buckets, Vec<f64>> {
                 256.0 * 1024.0,
             ]
         ),
+        (
+            Buckets::TransactionInsertionTimeInThreadPool,
```

Comment:
not for this PR, but I think having the name of the buckets implying their use is not optimal. That said, we discussed this already and IIRC we could not find a good naming convention :)